### PR TITLE
Fix type field in slack-invite.html

### PR DESCRIPTION
**Pull request by mribeirodantas**
- Add redirect page
- Fix type field for redirect page

### CODE REVIEW

File: content/slack-invite.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title=Slack invite redirect
+date=2016-07-26
+type=page
```
Comment (pditommaso) [inline content/slack-invite.html:3]: I think using `page` it adds in the resulting page the general HTML structure with menu, footer, etc. it does not?

Comment (mribeirodantas) [reply content/slack-invite.html:3]: Yes, it does. Ideally, we'd have to create a new light template for that, I think. If type is set to html, as it was in the first commit, jbake finds slack-invite.html, but does not create it in the output folder, so it throws Reason: Not Found if you try to access the URL.

Comment (pditommaso) [reply content/slack-invite.html]: Well spotted

**Review by pditommaso — APPROVED** [review]

**Comment by pditommaso** [comment]
Was this tested locally?

**Comment by mribeirodantas** [comment]
Yes, it was.

**Comment by pditommaso** [comment]
Awesome thanks

**Pull request merged**